harmony-api/src/services/postService.ts

```diff
@@ -27,6 +27,18 @@ export const createPost = async (postData?: Post): Promise<Post> => {
   return postResult
 }
 
+export const getPostByUserId = async (
+  userId?: string
+): Promise<Array<Post>> => {
+  if (!userId) {
+    return []
+  }
+
+  const posts = postRepo.getPostByUserId(userId)
+
+  return [...(await posts)]
```

Comment:
Not asking for changes here because I'm not that pedantic, but in the future you could write this as 
```suggestion
  return await postRepo.getPostByUserId(userId)
```